packages/dev/core/src/Events/pointerEvents.ts

```diff
@@ -109,21 +110,32 @@ export class PointerInfoPre extends PointerInfoBase {
  * The event member is an instance of PointerEvent for all types except PointerWheel and is of type MouseWheelEvent when type equals PointerWheel. The different event types can be found in the PointerEventTypes class.
  */
 export class PointerInfo extends PointerInfoBase {
+    private _pickInfo: Nullable<PickingInfo>;
+    private _inputManager: Nullable<InputManager>;
+
+    /**
+     * Defines the picking info associated to the info (if any)
+     */
+    public get pickInfo(): Nullable<PickingInfo> {
+        if (this._inputManager) {

```

Comment:
should it also check for pickinfo ? 

cause this.pickinfo could be set with this.inputmanager in the constructor ? nothing prevents it

Reply:
Changed logic